WordPress/Classes/Networking/AccountServiceRemoteREST.m

```diff
@@ -11,12 +18,12 @@ @implementation AccountServiceRemoteREST
 
 - (id)initWithApi:(WordPressComApi *)api
```

Comment:
I started to, but then remembered that `- (id)initWithApi:(WordPressComApi *)api` is declared in `ServiceRemoteRest.h`.  I'd want to match the implementation with its interface but I wouldn't want to introduce changes to `ServiceRemoteRest.h` that should be carried over to other things not strictly a part of this PR. 
Thoughts?